storybookjs/storybook
# Notifications placement

Issue:#5421 Notifications are obfuscated when the sidebar is narrow.


## What I did
- Made a new directory for notifications components
- Shortened update notification phrasing to "Storybook X.X.X is available"

**Todo**
My goal is to style the notifications like "flash" notifications, which appear above all other content. 
- Extract Notifications from `Sidebar` and put them at the top level of the layout. Currently app notifications live in the sidebar. 


## How to test
- Drag the sidebar to be narrow. If you don't see the update notification, ensure you're cache is cleared and that you're on an older version of SB.
